pnpm-workspace.yaml

```diff
@@ -37,9 +37,9 @@ catalog:
   '@unocss/reset': ^0.62.3
   '@unocss/runtime': ^0.62.3
   '@vitest/ui': ^2.0.5
-  '@vue/devtools-applet': 7.3.3
-  '@vue/devtools-core': 7.3.3
-  '@vue/devtools-kit': 7.3.3
+  '@vue/devtools-applet': 7.4.4
```

Comment:
Why do we pin the version instead of using the semver range `^` or `~`?

@webfansplz

Reply:
Because devtools was not stable enough in the past, there was a small range of breaking changes in minor releases. But now that I think it's relatively stable, I think we can let go of that limit, /cc @antfu